src/test/regress/isolation_schedule

```diff
@@ -89,11 +89,11 @@ test: isolation_ref_update_delete_upsert_vs_all_on_mx
 test: isolation_dis2ref_foreign_keys_on_mx
 test: isolation_metadata_sync_deadlock
 test: isolation_replicated_dist_on_mx
+test: isolation_multiuser_locking
```

Comment:
Good to see that we've enabled a new test for mx 👍  You might want to put it below isolation_replicate_reference_tables_to_coordinator to not to change shard ids on isolation_replicate_reference_tables_to_coordinator test.